### PR TITLE
13 baseline configurations fail to run

### LINKED ISSUE #13: Baseline Configurations Fail To Run
The baseline configurations currently do not run.  They encounter runtime errors that are caused by more than one issue:
* The setuptools dependency doesn't have a pkg_resources.packaging module
* The initialization parameters specified in the config don't get converted to the new simulator attributes

There could be other underlying issues

### PR DESCRIPTION
**Pull request by keatincf**
Updates to initializers and configurations to integrate with safe-autonomy-simulation

### CODE REVIEW

**Comment by keatincf** [comment]
I think these updates are ready to be merged in.  There is still an issue with multiagent translational inspection, both weighted and unweighted, where the episode starts in an unsafe state and ends early.  That seemed to be an existing issue from before the switch from safe-autonomy-dynamics to safe-autonomy-simulation and would require an update to the rejection sampler to handle checking a safety constraint for the collision of deputies.

Updating the rejection sampler might be out of scope of this issue.

**Comment by keatincf** [comment]
Looks like some unit tests need to be updated

File: safe_autonomy_sims/glues/rta_glue.py [hunk 1]
```diff
@@ -227,8 +227,8 @@ def get_observation(self, other_obs: OrderedDict, obs_space: OrderedDict, obs_un
 
     def get_info_dict(self):
         return {
-            "actual control": self.rta.control_actual,
-            "desired control": self.rta.control_desired,
+            "actual_control": self.rta.control_actual,
+            "desired_control": self.rta.control_desired,
```
Comment (jamie-cunningham) [inline safe_autonomy_sims/glues/rta_glue.py:231]: added underscore to suppress ray warning for logging

File: safe_autonomy_sims/platforms/cwh/cwh_platform.py
Comment (jamie-cunningham) [inline safe_autonomy_sims/platforms/cwh/cwh_platform.py:1]: The changed functions in this file reflect the current sa-simulation API which doesn't include ways to get relative values from entities themselves. That would involve some type of registering of the entities to each other or some notion of sensing. While this is a realistic constraint encouraged by CoRL's design, it seems out of scope and adds unnecessary complexity to these simulation environments. Instead, values are directly computed within the platform.

**Review by jamie-cunningham — APPROVED** [review]
The changes look good now. I added a few updates to address the current sa-simulation dependency's API changes. Note that the 6DOF environments are integrating to NaN solutions in sa-simulation which should either be addressed in that project or handled by sa-sims as a separate issue.

**Pull request merged**